app/demo-rv64-qemu-supervisor/app.toml

```diff
@@ -0,0 +1,36 @@
+name = "demo-rv64-qemu-supervisor"
+target = "riscv64imac-unknown-none-elf"
+board = "rv64-qemu-virt"
+chip = "../../chips/rv64-qemu-virt"
+stacksize = 2048
+
+[kernel]
+name = "demo-rv64-qemu-supervisor"
+requires = { flash = 20480, ram = 32768 }
+features = []
+
+[tasks.jefe]
+name = "task-jefe"
+priority = 0
+max-sizes = { flash = 16384, ram = 16384 }
+start = true
+features = ["semihosting-riscv", "log-stringbuf"]
```

Comment:
I don't think you can actually have both of these on at the same time.

Reply:
ACK. I'll drop stringbuf.